app/lib/dartdoc/handlers.dart

```diff
@@ -17,6 +17,7 @@ Future<shelf.Response> dartdocServiceHandler(shelf.Request request) async {
   final path = request.requestedUri.path;
   final handler = {
     '/debug': debugHandler,
+    '/robots.txt': rejectRobotsHandler,
```

Comment:
We might want to add a todo here about removing when this app has content...right?